features/between.feature

```diff
@@ -61,3 +61,21 @@ Feature: between
       | '<%=(Time.now - (5*60)).strftime('%H:%M:%S')%>'..'<%=(Time.now + (5*60)).strftime('%H:%M:%S')%>'  | '<%=(Time.now).strftime('%H:%M:%S')%>' | should     |
       | '<%=(Time.now + (5*60)).strftime('%H:%M:%S')%>'..'<%=(Time.now + (10*60)).strftime('%H:%M:%S')%>' | '<%=(Time.now).strftime('%H:%M:%S')%>' | should not |
 
+
+  Scenario Outline: Between ranges work inside of rule execution blocks
+    Given a rule template:
+      """
+      rule 'Testing Between Range' do
+        on_start
```

Comment:
Do we need to also test this on other types of trigger, at least a cron trigger?

Reply:
The every test is testing it with a cron trigger.

Reply:
I am referring to testing the between range. I'm not sure whether on_start runs in a different level of scope compared to the other triggers.